addon/-private/system/snapshot.js

```diff
@@ -347,6 +347,26 @@ Snapshot.prototype = {
   },
 
   /**
+    Serializes the snapshot using the serializer for the model.
+
+    Example
+
+    ```app/adapters/application.js
+    import DS from 'ember-data';
+
+    export default DS.Adapter.extend({
+      createRecord: function(store, type, snapshot) {
+        var data = this.serialize(snapshot, { includeId: true });
```

Comment:
Oops. I copied and pasted from the adapter docs and forgot to edit this line. Updated.